pkg/k8s/apis/cilium.io/utils/utils_test.go

```diff
@@ -7,30 +7,20 @@ import (
 	"fmt"
 	"testing"
 
-	. "github.com/cilium/checkmate"
+	"github.com/stretchr/testify/require"
 	"k8s.io/apimachinery/pkg/types"
 
-	"github.com/cilium/cilium/pkg/checker"
 	k8sConst "github.com/cilium/cilium/pkg/k8s/apis/cilium.io"
 	slim_metav1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
 	"github.com/cilium/cilium/pkg/labels"
 	"github.com/cilium/cilium/pkg/policy/api"
 )
 
-// Hook up gocheck into the "go test" runner.
-func Test(t *testing.T) {
-	TestingT(t)
-}
-
-type CiliumUtilsSuite struct{}
-
-var _ = Suite(&CiliumUtilsSuite{})
-
-func (s *CiliumUtilsSuite) Test_namespacesAreValid(c *C) {
-	c.Assert(namespacesAreValid("default", []string{}), Equals, true)
-	c.Assert(namespacesAreValid("default", []string{"default"}), Equals, true)
-	c.Assert(namespacesAreValid("default", []string{"foo"}), Equals, false)
-	c.Assert(namespacesAreValid("default", []string{"default", "foo"}), Equals, false)
+func Test_namespacesAreValid(t *testing.T) {
+	require.Equal(t, true, namespacesAreValid("default", []string{}))
```

Comment:
This comment is not in particular for this line but I am wondering why should we choose `require` over `assert` in that case. If the test fails on this line for example it will interrupt the test meaning we will have to rerun it to see the results from the line below...

Reply:
While assert allows the continuation of the test, it might just fail for other checks due to cascading reasons,  so IMO the benefit of having all the errors showing up become low.

There might be some other benefits/trade offs discussed before as per  below docs.

> Replace calls to c.Assert with require equivalents.

https://docs.cilium.io/en/stable/contributing/testing/unit/#migrate-gocheck